### PR TITLE
added scip (incl. soplex) and ceres-solver libs

### PR DESCRIPTION
**Pull request by soesau**
Added soplex, scip and ceres-solver to Ubuntu dockerfile
Allows testing of Polygonal Surface Reconstruction and mesh smoothing in PMP

### CODE REVIEW

File: Ubuntu/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     build-essential cmake \
     libopencv-dev \
     libsuitesparse-dev \
-    tar \
+    tar libgoogle-glog-dev \
```
Comment (sloriot) [inline Ubuntu/Dockerfile:9]: AFAIR, glog is an optional dependency of ceres. The pb of glog is libunwind, a (optional?) dependency, that cases huge slow down due to our usage of exception for filter failures.

Comment (sloriot) [reply Ubuntu/Dockerfile:9]: I added that here: https://doc.cgal.org/latest/Manual/thirdparty.html#title21
```
Attention
    Ceres indicates that glog is a recommended dependency. glog has libunwind as a recommended dependency. On some platforms, linking with libunwind was responsible for an increase of the runtime of the final application. If you experience such an issue, we recommand to compile Ceres without glog support.
```
I guess if you don't install it and give to cmake `-DMINIGLOG=ON` just to be safe that should be it.

Comment (soesau) [reply Ubuntu/Dockerfile:9]: I tried to build it without and it didn't work, if I remember correctly. I'll check again and use the -DMINIGLOG=ON.

Comment (soesau) [reply Ubuntu/Dockerfile]: Ok, works without glog by using `-DMINIGLOG=ON`

**Pull request merged**